src/domains/Directory.ts

```diff
@@ -7,13 +7,15 @@ export class Directory {
   description: string;
   createdAt: Date;
   updatedAt: Date;
+  emojiId: string;
   constructor({ _id, name, order, createdUser, isRoot, description, createdAt, updatedAt }: Directory) {
     this._id = _id;
     this.name = name;
     this.order = order;
     this.createdUser = createdUser;
     this.isRoot = isRoot;
     this.description = description;
+    this.emojiId = 'open_file_folder';
```

Comment:
ここは Directory domain のインスタンス化時に設定されるものなので 引数をそのまま設定したいですね

```suggestion
  constructor({ _id, name, order, createdUser, isRoot, description, emojiId, createdAt, updatedAt }: Directory) {
    this._id = _id;
    this.name = name;
    this.order = order;
    this.createdUser = createdUser;
    this.isRoot = isRoot;
    this.description = description;
    this.emojiId = emojiId;
```

Reply:
修正しました。